ci/terraform/oidc/account-interventions-alerts.tf

```diff
@@ -29,3 +29,35 @@ resource "aws_cloudwatch_metric_alarm" "account_interventions_p1_cloudwatch_alar
   alarm_description   = "${var.account_interventions_p1_alarm_error_threshold} or more Account Interventions errors have occurred in ${var.environment}.ACCOUNT: ${data.aws_iam_account_alias.current.account_alias}"
   alarm_actions       = [var.environment == "production" && var.account_intervention_service_abort_on_error ? data.aws_sns_topic.pagerduty_p1_alerts[0].arn : data.aws_sns_topic.slack_events.arn]
 }
+
```

Comment:
wonder if this sneaked in by mistake?

Reply:
Ah yes this commit @alhcomer pushed to the wrong branch by accident